index.html

```diff
@@ -7,12 +7,14 @@
     <link href="https://cdnjs.cloudflare.com/ajax/libs/flowbite/2.0.0/flowbite.min.css" rel="stylesheet" />
     <script src="https://cdn.tailwindcss.com"></script>
     <link rel="stylesheet" href="styles.css" />
+    <link rel="save.html" href="save.html"
```

Comment:
I don't think this is working, see line 16. The link should be in the body. In that case the link isn't working correctly and the link itself needs to be fixed.